src/app/pages/homepage/_homepage-theme.scss

```diff
@@ -16,10 +16,43 @@
 
     .docs-header-start {
       color: mat-color($primary);
+
+      .docs-link {
+        color: mat-color($primary, default-contrast);
+      }
     }
 
     .docs-homepage-row {
       color: mat-color($foreground, secondary-text);
     }
+
+    .docs-homepage-featured-components h1 {
+      color: mat-color($foreground, secondary-text);
+    }
+
+    a {
+      color: mat-color($foreground, secondary-text)
+    }
+
+    a.docs-link:hover {
+      color: mat-color($foreground, text);
+    }
+
+    .docs-homepage-guides-card-divider {
+      background: mat-color($accent)
+    }
+
+    [carousel-item]:hover {
+      .docs-guides-card.mat-card {
```

Comment:
Is the extra specificity here necessary? (i.e. could we just do `.docs-guides-card {`)